docs/agent.rst

```diff
@@ -48,54 +48,50 @@ Run an Agent
 ------------
 
 To run the *Agent* you should build it as indicated in :ref:`installation_label`.
-Once it is built successfully you just need to launch it executing the following command.
+Once it is built successfully, you just need to launch it executing the following command.
 
-For UDP communication: ::
+**UDP communication** ::
 
-    $ ./MicroXRCEAgent udp <port> [<discovery-port>]
+    $ ./MicroXRCEAgent --udp <local-port> [--discovery <discovery-port>] [--refs <refs-file>]
 
-For TCP communication: ::
+Launch a UDP server over port ``<local-port``. It has the following options:
 
-    $ ./MicroXRCEAgent tcp <port> [<discovery-port>]
+* ``--discovery <discovery-port>`` (only Linux): select a port for the discovery server which has the port number ``<7400>`` by default.
+* ``--refs <refs-file>``: load a reference file using the relative path to the working directory.
+  This reference file shall be composed by a set of Fast RTPS profile following the `XML syntax <https://eprosima-fast-rtps.readthedocs.io/en/latest/xmlprofiles.html>`_ describes in Fast RTPS.
+  The ``profile_name`` attribute of each profile represents a reference for an XRCE-Entity, so it could be used by the *Clients* in order to create entities by reference.
```

Comment:
```suggestion
  The ``profile_name`` attribute of each profile represents a reference to an XRCE-Entity so that it could be used by the *Clients* to create entities by reference.
```

Reply:
Done!